### PR TITLE
Existing RoomType Page + Navbar

### PR DESCRIPTION
**Pull request by kamilsmuga**
# 1. New Navbar
screen shot 2017-09-22 at 12 51 31 pm

# 2. Existing RoomType page
screen shot 2017-09-22 at 12 51 41 pm

# 3. Inheritance of "index.pug" across rooms pages

### CODE REVIEW

**Comment by tausifmuzaffar** [comment]
@kamilsmuga could component to span the full the page width?

**Comment by kamilsmuga** [comment]
@tausifmuzaffar like this? 
screen shot 2017-09-22 at 11 42 59 am

**Comment by BrandonMurnane** [comment]
personally id give it a little bit more padding

**Comment by kamilsmuga** [comment]
@tausifmuzaffar Implemented your feedback from the review. Rooms are displayed in separate tabs without "admin" container styling.

File: view/rooms/app.js [hunk 1]
```diff
@@ -13,14 +13,14 @@
  * to have their UI appended to the page.
  */
 
-const botsContainer = document.getElementById('botsContainer');
+const botsContainer = document.getElementById('roomsContainer');
```
Comment (iamigo) [inline view/rooms/app.js]: variable is named "botsContainer"?

File: view/rooms/utils/page.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    return document.getElementById(TITLE_ELEM_ID).innerHTML = title;
+  },
+
+  /**
+   * Sets page subtitle.
+   * @param  {Object} document - Document object
+   * @param  {String} title - Title string to be set
+   * @returns {String} - Title string if successfull
+   */
+  setSubTitle(subtitle) {
```
Comment (iamigo) [inline view/rooms/utils/page.js]: "subtitle" is an actual word so no camelcased "T" in the middle--should be `setSubtitle`

**Comment by kamilsmuga** [comment]
@iamigo changes implemented (f5cf52f)

File: view/rooms/utils/page.js [hunk 3]
```diff
@@ -34,10 +34,10 @@ module.exports = {
   /**
    * Sets page subtitle.
    * @param  {Object} document - Document object
-   * @param  {String} title - Title string to be set
-   * @returns {String} - Title string if successfull
+   * @param  {String} subtitle - Title string to be set
+   * @returns {String} - subtitle string if successfull
```
Comment (iamigo) [inline view/rooms/utils/page.js]: `successfull` should be `successful`

**Pull request merged**